docs/book/drafts/spaces-sdks-docker-zenml.md

```diff
@@ -0,0 +1,157 @@
+the screen. This should look something like this:
+`https://<YOUR_USERNAME>-<SPACE_NAME>.hf.space`.
+
+You can now use this URL to connect to your ZenML server from your local machine
+with the following CLI command (after installing ZenML, and using your custom
+URL instead of the placeholder):
+
+```shell
+zenml connect --url '<YOUR_HF_SPACES_DIRECT_URL>' --no-verify-ssl --username='default'
+```
+
+You can also use the Direct URL in your browser to use the ZenML dashboard as a
+fullscreen application (i.e. without the HuggingFace Spaces wrapper around it).
+
+## Extra Configuration Options
+
+By default the ZenML application will be configured to use a SQLite
+non-persistent database. If you want to use a persistent database, you can
+configure this by amending the `Dockerfile` in your Space's root directory. For
+full details on the various parameters you can change, see [our reference
+documentation](./docker.md#zenml-server-configuration-options) on configuring
+ZenML when deployed with Docker.
+
+{% hint style="info" %}
+If you are using the space just for testing and experimentation, you don't need
+to make any changes to the configuration. Everything will work out of the box.
+{% endhint %}
+
+You can also use an external secrets backend together with your HuggingFace
+Spaces as described in [our
+documentation](./docker.md#zenml-server-configuration-options). You should be
+sure to use HuggingFace's inbuilt 'Repository secrets' functionality to
+configure any secrets you need to use in your`Dockerfile` configuration. [See the
+documentation](https://huggingface.co/docs/hub/spaces-sdks-docker#secret-management)
+for more details how to set this up.
+
+{% hint style="warning" %}
+If you wish to use a cloud secrets backend together with ZenML for secrets
+management, **you must update your password** on your ZenML Server on the
+Dashboard. This is because the default user created by the
+HuggingFace Spaces deployment process has no password assigned to it and as the
+Space is publicly accessible (since the Space is public) *potentially anyone
+could access your secrets without this extra step*. To change your password
+navigate to the Settings page by clicking the button in the upper right hand
+corner of the Dashboard and then click 'Update Password'.
+{% endhint %}
+
+
+## Upgrading your ZenML Server on HF Spaces
+
+The default space will use the latest version of ZenML automatically. If you
+want to update your version, you can simply select the 'Factory reboot' option
+within the 'Settings' tab of the space. Note that this will wipe any data
+contained within the space and so if you are not using a MySQL persistent
+database (as described above) you will lose any data contained within your ZenML
+deployment on the space. You can also configure the space to use an earlier
+version by updating the `Dockerfile`'s `FROM` import statement at the very top.
+
+## Next Steps
+
+As a next step, check out our [ZenBytes Tutorial
```

Comment:
Id suggest people to go to the docs starter guide here instead as we might deprecate ZenBytes in the near future in favor of docs (Something in the works)